### PR TITLE
Syringe changes.

### PR DESCRIPTION
**Pull request by QuietlyWhisper**
## About the PR

Debuff of delay per 5u reduced from 1s to 0.5s
Delay on crit patients reduced by 25%

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Whisper
- tweak: Debuff of delay per 5u reduced from 1s to 0.5s
- tweak: Delay on crit patients reduced by 25%

### CODE REVIEW

**Comment by QuietlyWhisper** [comment]
Test fail seems unrelated to my PR.

**Comment by asperger-sind** [comment]
I don't think inceasing the self-inject/draw delay even more is a good idea, it already takes a while and nobody does it as-is; why make it even worse?

**Comment by QuietlyWhisper** [comment]
> I don't think inceasing the self-inject/draw delay even more is a good idea, it already takes a while and nobody does it as-is; why make it even worse?

See; #9697

File: Content.Server/Chemistry/EntitySystems/ChemistrySystem.Injector.cs [hunk 1]
```diff
@@ -229,8 +229,8 @@ private void InjectDoAfter(InjectorComponent component, EntityUid user, EntityUi
 
         var actualDelay = MathF.Max(component.Delay, 1f);
 
-        // Injections take 1 second longer per additional 5u
-        actualDelay += (float) component.TransferAmount / component.Delay - 1;
+        // Injections take 0.5 seconds longer per additional 5u
+        actualDelay += (float) component.TransferAmount / component.Delay - 0.5f;
```
Comment (rafucuk) [inline Content.Server/Chemistry/EntitySystems/ChemistrySystem.Injector.cs:233]: I don't think decreasing time will affect that much

Comment (QuietlyWhisper) [reply Content.Server/Chemistry/EntitySystems/ChemistrySystem.Injector.cs:233]: Not really; but you're looking at a difference of 3 -> 1.5 seconds on 15u
It mostly only affects large doses from the syringe.

**Comment by deltanedas** [comment]
changelog doesnt mention syringes

**Comment by QuietlyWhisper** [comment]
uh oh

**Pull request merged**